### PR TITLE
change cost of policy-generated actions to zero in sesame

### PR DESCRIPTION
**Pull request by tomsilver**
see the new comment for justification. this is important for @Alchemy5 's code and I suspect that it will also improve @wester-is-not-real 's results.

@wester-is-not-real  -- we should regenerate the pg4 results after this is merged.

### CODE REVIEW

**Review by ronuchit — COMMENTED** [review]
looks reasonable. is this what's already happening in the utils.py functions used by PG3, e.g. `run_policy_guided_astar`?

is this `cumulative_cost=0` the same functional thing as `rollout_step_cost` being 0 in that function?

**Comment by tomsilver** [comment]
@ronuchit thanks for looking, yeah exactly

File: tests/test_planning.py [hunk 1]
```diff
@@ -514,7 +514,7 @@ def test_policy_guided_sesame():
     pick_nsrt = nsrt_name_to_nsrt["Pick"]
     place_nsrt = nsrt_name_to_nsrt["Place"]
 
-    # When using a perfect policy, we should only expand nodes along the plan.
+    # When using a perfect policy, we should only expand the first node.
```
Comment (ronuchit) [inline tests/test_planning.py:517]: i don't get it, why did this change? how can you not expand all the nodes to the goal?

**Review by ronuchit — APPROVED** [review]

File: tests/test_planning.py (hunk 1 above)
Comment (tomsilver) [reply tests/test_planning.py:517]: we create all the nodes to the goal, but we don't need to expand them

**Pull request merged**